docs/manual/forms/examples/repeating_groups.rst

```diff
@@ -0,0 +1,114 @@
+
+Formulier maken
+===============
+
+#. Maak een formulier aan met de volgende gegevens:
+
+   * **Naam**: Herhalende groep demo
+
+#. Klik op het tabblad **Stappen en velden**.
+#. Klik aan de linkerkant op **Stap toevoegen** en selecteer **Maak een nieuwe
+   formulierdefinitie**.
+#. Onder de sectie **(Herbruikbare) stapgegevens** vul het volgende in:
+
+   * **Naam**: Stap met herhalende groep
+
+#. Scroll naar de sectie **Velden** en klik op **Speciale velden**.
+#. Sleep een **Herhalende groep** component op het witte vlak, vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Basis** > **Label**: _`Auto's`
+   * **Weergave** > **Open eerste groep als leeg**: *aangevinkt*
+
+#. Klik op **Formuliervelden** en sleep een **Tekstveld** component binnen de herhalende groep. Vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Kleur
+
+#. Sleep een **Bedrag** component binnen de herhalende groep. Vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Waarde
+
+#. Klik op **Opmaak** en sleep een vrije tekst component op het witte vlak. Vul de volgende gegevens in:
+
+   * Onder **Vrije tekst component**: ``De totale waarde van uw auto's is: €{{ totaleWaarde|localize }}``
+   * **Verborgen**: *aangevinkt*
+
+#. Druk daarna op **Opslaan en opnieuw bewerken**.
+#. Klik op het tabblad **Variabelen** en daarna op het tabblad **User defined**.
+#. Voeg een variabele toe met de volgende gegevens:
+
+   * **Naam**: Aantal autos
+   * **Data type**: float
+
+#. Voeg nog een variabele toe met de volgende gegevens:
+
+   * **Naam**: Totale waarde
+   * **Data type**: float
+
+#. Klik op het **Logica** tabblad en voeg een geavanceerde regel met de de volgende gegevens:
+
+   * Trigger: ``{"!!": [true]}``
+   * Acties:
+
+     * **Wijzig de waarde van een variabele/component** > **Totale waarde** >
+
+     .. code-block:: json
+
+        {
+            "reduce": [
+                {"var": "autos"},
+                {"+": [{"var": "accumulator"}, {"var": "current.waarde"}]},
+                0
+            ]
+        }
+
+     .. note::
+
+        De variabele ``autos`` moet overeenkomen met de eigenschapsnaam van het :ref:`herhalende groep component<Auto's>`.
+        De variabelen ``accumulator`` en ``current`` refereren binnen de ``reduce`` functie naar respectievelijk de
+        resultaatwaarde en het huidige groep.
+        Binnen een groep kunnen we weer verwijzen naar een individueel component.
+        Hier wordt dat bijvoorbeeld gedaan naar het component met de eigenschapsnaam ``waarde``.
+
+     * **Wijzig de waarde van een variabele/component** > **Aantal autos** >
+
+     .. code-block:: json
+
+        {"reduce": [{"var": "autos"}, {"+": [{"var": "accumulator"}, 1]}, 0]}
+
+    .. note::
+
+       Hier wordt bijna hetzelfde gedaan als hierboven, maar in plaats van de ``waarde`` tellen we nu steeds het
+       getal ``1`` op per iteratie om tot een totaal aantal te komen.
```

Comment:
```suggestion
       getal ``1`` op per iteratie om tot een totale aantal te komen.
```

Reply:
I would make it "het totale aantal" or "een totaalaantal" :p

https://www.villamedia.nl/artikel/taaltip-het-totaal-aantal-totale-aantal-totaalaantal-deelnemers

Reply:
Ok! Went for het totale aantal :upside_down_face: